apps/developer-portal/src/components/remote-repository-configuration/remote-repository-list.tsx

```diff
@@ -0,0 +1,156 @@
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { InterfaceRemoteRepoConfig } from "../../models";
+import { ResourceList, ResourceListItem, EmptyPlaceholder, PrimaryButton, ConfirmationModal } from "@wso2is/react-components";
+import { UIConstants, AppConstants } from "../../constants";
+import { useTranslation, Trans } from "react-i18next";
+import { Icon } from "semantic-ui-react";
+import { OverviewPageIllustrations } from "../../configs";
+import { history } from "../../helpers";
+
+interface RemoteRepoListProp {
+    repoObjectList: InterfaceRemoteRepoConfig[];
+    handleConfigDelete: (repoConfig: InterfaceRemoteRepoConfig) => void;
+    showCreateWizard: (state: boolean) => void;
+}
+
+export const RemoteRepoList: FunctionComponent<RemoteRepoListProp> = (props: RemoteRepoListProp): ReactElement => {
+
+    const { t } = useTranslation();
+
+    const { repoObjectList, handleConfigDelete, showCreateWizard } = props;
+
+    const [ currentDeleteConfig, setCurrentDeleteConfig ] = useState<InterfaceRemoteRepoConfig>();
+    const [ showRoleDeleteConfirmation, setShowDeleteConfirmationModal ] = useState<boolean>(false);
+
+    /**
+     * Redirects to the identity provider edit page when the edit button is clicked.
+     *
+     * @param {string} idpId Identity provider id.
+     */
+    const handleIdentityProviderEdit = (configId: string): void => {
+        history.push(AppConstants.PATHS.get("REMOTE_REPO_CONFIG_EDIT").replace(":id", configId));
+    };
+
+    /**
+     * Resolve the relevant placeholder.
+     *
+     * @return {React.ReactElement}
+     */
+    const showPlaceholders = (): ReactElement => {
+        if (!repoObjectList || repoObjectList?.length === 0) {
+            return (
+                <EmptyPlaceholder
+                    action={
+                        <PrimaryButton onClick={ () => { showCreateWizard(true) } }>
+                            <Icon name="add"/>
+                            { t("devPortal:components:remoteConfig:placeholders.emptyList.action") }
+                        </PrimaryButton>
+                    }
+                    title={ t("devPortal:components:remoteConfig:placeholders.emptyList.title") }
+                    subtitle={ [
+                        t("devPortal:components:remoteConfig:placeholders.emptyList.subtitles.0"),
+                        t("devPortal:components:remoteConfig:placeholders.emptyList.subtitles.1"),
+                        t("devPortal:components:remoteConfig:placeholders.emptyList.subtitles.2")
+                    ] }
+                    image={ OverviewPageIllustrations.statsOverview.idp }
```

Comment:
Better to export a separate variable and use here rather than using `OverviewPageIllustrations` which is meant for overview page.

Reply:
Fixed with commit 8c08fbd